client/src/components/MyAccount/AddItemFormSteps/StepThree.jsx

```diff
@@ -31,6 +38,20 @@ const StepThree = ({
     });
   }, [user]);
 
+  useEffect(() => {
+    (async () => {
+      const response = await getUserCard(user.id);
```

Comment:
`const { data } = await getUserCard(user.id);`, you can destructure response right on declaration